.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+*.zip
```

Comment:
You definitely added this after commiting your zip file. When that happens you can use `git rm` to actually remove the file from the repo, then ignore will make sure you never commit it after that. If the file has already been committed, .gitignore won't remove it.